app/components/Sidebar/components/Transition.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
```

Comment:
I like the effect here, but it's causing a lot of small display problems due to the `overflow: hidden`. Let's ship this PR without the animated transition aspect and we can handle adding a transition in a separate change set

I can detail the issues there